crates/sc-consensus-subspace-rpc/src/lib.rs

```diff
@@ -154,6 +155,16 @@ pub trait SubspaceRpcApi {
 
     #[method(name = "subspace_lastSegmentHeaders")]
     async fn last_segment_headers(&self, limit: u64) -> Result<Vec<Option<SegmentHeader>>, Error>;
+
+    /// Block/transaction archived object mappings subscription
+    #[subscription(
+        name = "subspace_subscribeArchivedObjectMappings" => "subspace_archived_object_mappings",
+        unsubscribe = "subspace_unsubscribeArchivedObjectMappings",
+        item = GlobalObject,
```

Comment:
Since JSON is self-describing type, we'll probably want to switch this to a tuple or even hex-encoded binary representation, otherwise a significant amount of bandwidth will be wasted here assuming we expect a lot of objects here.

Reply:
I'd like to do this later, to make debugging easier. I've put it on the list in the Notion doc.